code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -264,3 +264,278 @@
 
 #undef RIGHT_HAND
 #undef LEFT_HAND
+
+/obj/item/organ/internal/cyberimp/arm/katana
+	name = "dark shard"
+	desc = "An eerie metal shard surrounded by dark energies."
+	icon = 'icons/obj/lavaland/artefacts.dmi'
+	icon_state = "cursed_katana_organ"
+	status = 0
+	contents = newlist(/obj/item/cursed_katana)
```

Comment:
should probably add this item in the Initialize sequence

Reply:
Handled as all other arm implants are